pdg/src/readers/ObjectReader.cpp

```diff
@@ -95,9 +95,9 @@ void ObjectReader::putInHand(struct objectIn_t& objectIn, string id, MovableObje
   {
     bool addFactHand = true;
     if (!putAtJointPosition(object, objectIn.Agent_[id], objectIn.Hand_[id],
-                            list_msg.human_msg, true)) // try to put in human
+                            list_msg.human_msg, false)) // try to put in human, why should we set it as simu ?
```

Comment:
This is from 4 years, but I would say,  because it's not from observation?

Reply:
This is what I thought first, but when we set it to true, it updates the pose of the object in toaster_simu. This means that it assumes that all objects are simulated... However, setting it to false (as I did) also raises a small problem, it is that if we try to control the simulated object with the keyboard after remove_from_hand, it will start at the previous position.

I don't really know what is the best thing to do